### DISCUSSION TITLE
Only dark theme without toggle option

### DISCUSSION THREAD

**Discussion by Aksheet10** [smaranjitghose/awesome-portfolio-websites#1065] [General]
If you dont want light theme and the toggle bar and all pages as dark theme follow these steps

1. In all the `.html` pages add this line as the second line `<html lang="en", light-mode="dark">`
 
![image](https://user-images.githubusercontent.com/71016915/133873486-2755ac47-2192-4c58-aee5-5ca32a32e966.png)

Now all your pages will be dark theme only by default.

---

## Toggle bar removing - 

Open `assets/js/app.js`

Completely comment out or remove the `function toggle_light_mode()`

![image](https://user-images.githubusercontent.com/71016915/133873650-1f90034b-adb6-4413-9c83-b86eed112b16.png)

Then search for this line in app.js

`<input type="checkbox" id="dark_toggler" class="dark_toggler" aria-label="Toggle Light Mode" onclick="toggle_light_mode()">`

Comment it or remove it.

At this stage your website will look like this

![image](https://user-images.githubusercontent.com/71016915/133873776-1e20e299-11d1-454f-8c17-15158f4fd19c.png)

Now go line 4 in app.js (the `let header =$`)  part and then add the `dark-theme` in it.
So it should look like - `<nav class="navbar navbar-expand-lg navbar-light fixed-top dark-theme" id="navbar">`

Now your website should look like this

![image](https://user-images.githubusercontent.com/71016915/133873837-1c52835e-0645-49ee-a831-6363546dead5.png)

**Comment by FoggY9** [comment]
link icons are still white

**Comment by anushbhatia** [comment]
dark mode is not properly functional because color schemes are still are in development so we haven't done that.
